optimum/exporters/onnx/__main__.py

```diff
@@ -36,7 +36,7 @@ def main():
     )
     parser.add_argument(
         "--task",
-        default="default",
+        default="auto",
         help="The type of tasks to export the model with.",
```

Comment:
outside the scope of this PR but

`The type of task` (singular, not plural), no?